storm-core/src/clj/backtype/storm/daemon/worker.clj

```diff
@@ -114,12 +114,38 @@
   (fast-list-iter [[task tuple :as pair] tuple-batch]
     (.serialize serializer tuple)))
 
+(defn- mk-backpressure-handler [executors]
+  "make a handler that checks and updates worker's backpressure flag"
+  (disruptor/backpressure-handler 
+    (fn [worker]
+      (let [storm-id (:storm-id worker)
+            assignment-id (:assignment-id worker)
+            port (:port worker)
+            storm-cluster-state (:storm-cluster-state worker)
+            prev-backpressure-flag @(:backpressure worker)]
+        (doseq [ed executors] ;; Debug, TODO: delete
+          (log-message "zliu executor" (.get-executor-id ed) " flag is " (.get-backpressure-flag ed)))
```

Comment:
Thanks for the reminder, I will remove all those debug messages soon after the unit test commit.